addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -220,18 +220,20 @@ body .modal {
                 min-height: 300px;
             }
 
-            .o_we_images > .o_existing_attachment_cell {
-                // give enough space for the hover buttons
-                min-width: 64px;
-                max-width: 256px;
-                height: 192px;
-                img {
-                    max-height: 100%;
-                }
+            .o_we_images > .o_existing_attachment_cell img {
+                max-width: 100%;
+                background-image: url("/web_editor/static/src/img/transparent.png");
+            }
+
+            .o_we_attachment_placeholder {
+                // These are 3/2 * ImageWidget.MIN_ROW_HEIGHT, since that's the aspect ratio of most photos.
+                flex-grow: 192;
```

Comment:
flex-grow: 192?

Reply:
As noted by the comment: `3/2 * ImageWidget.MIN_ROW_HEIGHT, since that's the aspect ratio of most photos`

The new layout system (ab?)uses the flex-grow property: for images it's set to `ImageWidget.MIN_ROW_HEIGHT * aspectRatio`, this allows the images to grow in proportion to their widths, which maintains their height to all be the same on one row. The placeholder simulate additional photos at the end, so that if there is a single image on the last row it doesn't take up the entire width, but behaves as though there are more photos after it.

Reply:
Yep, like as said, did not check the way it worked yet, that's cool :)
Note, I am never sure what I prefer when it comes to having scss value related to a JS value. Maybe in this case, I won't mind having those rules set as inline style from the JS, using the JS value. What do you think?

Btw:
```xml
<t t-foreach="new Array(20).fill()">
```
=>
```xml
<t t-foreach="20">
```

Reply:
Probably better to set it in JS if possible.